crates/next-core/js/src/entry/next-hydrate.tsx

```diff
@@ -20,8 +20,19 @@ import * as page from ".";
     assetPrefix,
   });
 
+  const pagePath = window.__NEXT_DATA__.page;
+  window.__BUILD_MANIFEST = {
+    [pagePath]: [],
+    __rewrites: {
+      beforeFiles: [],
+      afterFiles: [],
+      fallback: [],
+    } as any,
```

Comment:
Was there an issue typing this properly?

Reply:
yes, `window.__BUILD_MANIFEST` is defined as `Record<string, string[]>`